src/main/java/io/cryostat/targets/Target.java

```diff
@@ -72,9 +72,7 @@ public class Target extends PanacheEntity {
     @NotNull
     public URI connectUrl;
 
-    @Column(unique = true)
-    @NotBlank
-    public String alias;
+    @Column @NotBlank public String alias;
```

Comment:
I think `@Column` can be dropped if we aren't specifying any non-default options. It doesn't really matter, but it's consistent with the style where other fields aren't annotated with it if they don't need it.